src/example-types/tagsQuery.js

```diff
@@ -0,0 +1,97 @@
+let _ = require('lodash/fp')
+let { parens, quote } = require('futil-js')
+let Combinatorics = require('js-combinatorics')
+
+// Split text into words and return array of string permutations
+let wordPermutations = _.flow(
+  _.split(/\s+/),
+  x => Combinatorics.permutation(x).toArray(),
+  _.map(_.join(' '))
+)
+
+let quoteAndTilde = _.curry(
+  (tag, text) =>
+    (tag.isPhrase ? quote(text) : text) +
+    (tag.misspellings || tag.distance ? '~' : '') +
```

Comment:
@daedalus28 Should we consider changing misspellings behavior to use `~1` instead of `~`? https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-query-string-query.html#_fuzziness

Reply:
That's a great idea 👍  Let's include that link in the source as well to explain why